internal/service/alarms/serve.go

```diff
@@ -138,47 +148,10 @@ func Serve() error {
 		return fmt.Errorf("error starting server: %w", err)
 	case <-ctx.Done():
 		slog.Info("Shutting down server")
-		if err := gracefulShutdown(srv); err != nil {
-			return err
+		if err := common.GracefulShutdown(srv); err != nil {
```

Comment:
let's simply return here. No need to wrap the error again in my opinion

Reply:
Our linter configuration requires that we wrap this error since they are in different packages.